kustomize/kustomization.yaml

```diff
@@ -27,3 +27,4 @@ components:
 # - components/container-images-tag-suffix
 # - components/container-images-registry
 # - components/native-grpc-health-check
+# - components/without-loadgenerator
```

Comment:
I think it would be useful to have `without-loadgenerator` as a Kustomize Component — more accessible to anyone using Online Boutique.
So I've turned it into a `kind: Component` and moved it to `/kustomize/components` with a [`README.md`](https://github.com/GoogleCloudPlatform/microservices-demo/tree/minherz/implement-1478/kustomize/components/without-loadgenerator#readme).

Reply:
good catch